### ISSUE TITLE
Check for any accessibility issues

### ISSUE THREAD

**Issue by lukas-wg2** [working-group-two/wgtwo.com#38]
Chrome have a neat tool to check for any CSS issues on a webpage, probably a good idea to check for potential contrast issues.
![Screenshot from 2022-02-17 12-04-51](https://user-images.githubusercontent.com/43165886/154469216-1c4f5d77-d746-45bb-821b-8c02e4a37e72.png)
![image](https://user-images.githubusercontent.com/43165886/154469555-5aaa97cd-79d5-48ca-9dd8-741752fd1b21.png)
![image](https://user-images.githubusercontent.com/43165886/154469057-99a95bb0-f854-45eb-bb4e-997b7195215b.png)

**Comment by s-utstrand** [comment]
Are there any specific concerns regarding accessibility besides color contrast? I've run the WCAG color contrast checker, and all AA checks are passed (the two failing are the footer logo, where the checker thinks the font color is black, plus a hidden element). 
![image](https://user-images.githubusercontent.com/104059760/166903802-05803742-e2a9-43a1-9419-f3a68980d825.png)

I have also tested keyboard navigation, which works fine.

So, in total, I would say the website is well inside the requirements for accessibility.

**Comment by torotimes** [comment]
@lukas-wg2 - please review input from @s-utstrand. We can close the issue if you're comfortable?

**Comment by lukas-wg2** [comment]
Yes, I'll close it! :)
